src/game/client.cpp

```diff
@@ -465,7 +465,7 @@ namespace client
 
     void setloadweap(const char *list)
     {
-        vector<int> items;
+        std::vector<int> items;
```

Comment:
What's the motivation for this change?

Reply:
IIRC there was a compiler warning in the `sendf` from aiman.h reviewed by @robalni related to the custom vector's `getbuf()`, so here we are